js/src/product-feed/product-statistics/status-box.js

```diff
@@ -85,14 +85,12 @@ function getSyncResult( {
 	};
 }
 
-const issuesQuery = {
-	page: 1,
-	per_page: ISSUE_TABLE_PER_PAGE,
-};
-
 function FeedStatus() {
-	const { data } = useAppSelectDispatch( 'getMCIssues', issuesQuery );
-	const status = getUnsolvedStatusText( data?.total );
+	const issues = useMCIssues();
+	const status = getUnsolvedStatusText(
+		issues[ ISSUE_TYPE_ACCOUNT ]?.data?.total +
+			issues[ ISSUE_TYPE_PRODUCT ]?.data?.total
+	);
```

Comment:
We just try to sum the totals here as before. Notice `getUnsolvedStatusText` handles NaN cases by its own.